src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -507,4 +697,29 @@ object BaggedResult {
     }
     scores.map(Math.max(floor, _))
   } ensuring (vec => vec.forall(_ >= 0.0))
+
+  /**
+    * Calculate the correlation coefficient given a sequence of covariance scores for each training point.
+    * In theory, the covariance is the sum of the individual covariance scores. But because each covariance score
+    * is noisy, the resulting sum can be larger in absolute value than `sigmaX * sigmaY`, which would translate into
```

Comment:
This comment motivates the purpose of the method, but it would be helpful to add a sentence saying what the consequence is on the resulting sum.